### PR TITLE
Simulation fixes

### PR DESCRIPTION
**Pull request by rhuairahrighairidh**

### CODE REVIEW

**Comment by denalimarsh** [comment]
@rhuairahrighairidh what's the status of this PR?

**Comment by rhuairahrighairidh** [comment]
I've tidied this PR for review.
It contains a couple of fixes for issues that came up infrequently in simulations, plus some tidying of the simulation dockerfile.

File: simulations/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This can run sims but doesn't collect the results. This is handled by a custom s
 
 ## Running sims and uploading to S3
 
-The dockerfile in this repo defines the docker image to run sims. It's just a normal app, but with the aws cli included, and the custom script.
+The dockerfile in this repo defines the docker image to run sims. It includes kava source, and the aws cli, and the custom script.
```
Comment (karzak) [inline simulations/README.md:24]:
```suggestion
The dockerfile in this repo defines the docker image to run sims. It includes the kava source code, aws cli, and the custom simulation script.
```

File: x/cdp/simulation/operations.go [hunk 2]
```diff
@@ -172,7 +174,8 @@ func SimulateMsgCdp(ak types.AccountKeeper, k keeper.Keeper, pfk types.Pricefeed
 
 			_, _, err := app.Deliver(tx)
 			if err != nil {
-				return simulation.NoOpMsg(types.ModuleName), nil, err
+				// to aid debugging, add the stack trace to the comment field of the returned opMsg
+				return simulation.NewOperationMsg(msg, false, fmt.Sprintf("%+v", err)), nil, err
```
Comment (karzak) [inline x/cdp/simulation/operations.go:178]: Mind adding this to all the modules? I've found the lack of logs when there's an error after `app.Deliver` to be a source of frustration.

Comment (rhuairahrighairidh) [reply x/cdp/simulation/operations.go:177]: added

**Pull request merged**